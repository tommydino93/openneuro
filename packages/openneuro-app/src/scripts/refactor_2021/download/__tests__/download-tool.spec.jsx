import React from 'react'
import { shallow } from 'enzyme'
import DownloadTool from '../download-tool.jsx'

const defProps = {
  match: {
    params: {
      datasetId: 'ds000224',
      snapshotId: '1.0.0',
    },
  },
}

describe('dataset/download/DownloadTool', () => {
  it('renders successfully', () => {
    const wrapper = shallow(<DownloadTool {...defProps} />)
    expect(wrapper).toMatchSnapshot()
  })
})
