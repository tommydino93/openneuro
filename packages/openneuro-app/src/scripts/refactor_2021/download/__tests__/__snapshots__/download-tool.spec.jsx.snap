// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dataset/download/DownloadTool renders successfully 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
